pycnn.py

```diff
@@ -67,6 +96,20 @@ def validate(self, input_location):
     # tempA: control template, tempB: feedback template
     def imageprocessing(self, inputlocation, outputlocation,
                         tempA, tempB, initialcondition, Ib, t):
+        """Process the image with the input arguments.
+
+        Args:
+            inputlocation (str): The string path for the input image.
+            outputlocation (str): The string path for the output processed
+                image.
+            tempA (:obj:`list` of :obj:`list`of :obj:`float`): The template
```

Comment:
Please refer tempA (template A) as the feedback template and tempB (template B) as control template everywhere.